tests/feature/CountryTest.php

```diff
@@ -15,7 +18,43 @@ class CountryTest extends TestCase
      */
     public function itCanIndex()
     {
-        $country = factory(Country::class, 1)->make(['name' => 'nl', 'label' => 'Netherlands']);
-        $this->index('countries.index', CountryResource::make($country));
+        $countries = Country::all();
+        $this->index('countries.index', CountryResource::collection($countries));
+    }
+
+    /**
+     * @test
+     */
+    public function itCanShow()
+    {
+        $country = factory(Country::class)->create();
+        $this->show('countries.show', CountryResource::make($country));
+    }
+
+    /**
+     * @test
+     */
+    public function itCanStore()
+    {
+        $country = factory(Country::class)->create();
+        $this->store('countries.store', CountryResource::make($country), $country->toArray());
+    }
+
+    /**
+     * @test
+     */
+    public function itCanUpdate()
+    {
+        $country = factory(Country::class)->create();
+        $this->update('countries.update', CountryResource::make($country), $country->toArray());
```

Comment:
you are updating nothing, because you are using the same country that is already created in the post should be something like:
$country = factor()->create()
$update = factory()->make()